docs/guides/chip_tool_guide.md

```diff
@@ -1013,3 +1021,234 @@ In this command:
+    | * operating-mode                                                                    |
+    | * supported-operating-modes                                                         |
+    | * default-configuration-register                                                    |
+    | * enable-local-programming                                                          |
+    | * enable-one-touch-locking                                                          |
+    | * enable-inside-status-led                                                          |
+    | * enable-privacy-mode-button                                                        |
+    | * local-programming-features                                                        |
+    | * wrong-code-entry-limit                                                            |
+    | * user-code-temporary-disable-time                                                  |
+    | * send-pinover-the-air                                                              |
+    | * require-pinfor-remote-operation                                                   |
+    | * expiring-user-timeout                                                             |
+    | * generated-command-list                                                            |
+    | * accepted-command-list                                                             |
+    | * event-list                                                                        |
+    | * attribute-list                                                                    |
+    | * feature-map                                                                       |
+    | * cluster-revision                                                                  |
+    +-------------------------------------------------------------------------------------+
+    ```
+
+1. Add the argument of your choice to the subscription command, using the
+   following pattern:
+
+    ```bash
+    >>> <cluster-name> subscribe <argument> <min-interval> <max-interval> <node_id> <endpoint_id>
+    ```
+
+    In this command:
+
+    - _<cluster-name\>_ is the name of the cluster.
+    - _<argument\>_ is the name of the chosen argument.
+    - _<min-interval\>_ specifies the minimum number of seconds that must elapse
+      since the last report for the server to send a new report.
+    - _<max-interval\>_ specifies the number of seconds that must elapse since
+      the last report for the server to send a new report.
+    - _<node-id\>_ is the user-defined ID of the commissioned node.
+    - _<endpoint_id\>_ is the ID of the endpoint where the chosen cluster is
+      implemented.
+
+    For example:
+
+    ```bash
+    >>> doorlock subscribe lock-state 5 10 1 1
+    ```
+
+    > **Tip:** You can also set the `$NODE_ID` and `$ENDPOINT_ID` variables for
+    > the _<node-id\>_ and _<endpoint_id\>_, respectively, and use them in your
+    > commands.
```

Comment:
I don't think you can, in interactive mode... Sorry, my comments yesterday were very unclear on this: it would work at the shell as the original command line there had it, but inside interactive mode there is no shell variable expansion.